foodsaving/users/models.py

```diff
@@ -134,21 +143,69 @@ def _send_welcome_mail(self):
         ).send()
 
     @transaction.atomic
-    def send_new_verification_code(self):
+    def send_mail_verification_code(self):
         self._unverify_mail()
         prepare_email('send_new_verification_code', self, {
-            'url': '{hostname}/#/verify-mail?key={code}'.format(
+            'url': '{hostname}/#/verify_mail?code={code}'.format(
                 hostname=settings.HOSTNAME,
                 code=VerificationCode.objects.get(user=self, type=VerificationCode.EMAIL_VERIFICATION).code
             )
         }, to=self.unverified_email).send()
 
+    @transaction.atomic
+    def send_account_deletion_verification_code(self):
+        VerificationCode.objects.filter(user=self, type=VerificationCode.ACCOUNT_DELETE).delete()
+        prepare_email('accountdelete_request', self, {
+            'url': '{hostname}/#/delete_account?code={code}'.format(
```

Comment:
At some point, I'd like to put all frontend urls into a file that makes it possible to reuse them (well, at least the ones that we use in the backend). Might also help if we change something about the frontend routing in future.